app/src/main/java/edu/grinnell/kdic/RadioService.java

```diff
@@ -214,17 +208,17 @@ public void play() {
             if (result != AudioManager.AUDIOFOCUS_REQUEST_GRANTED) {
                 // could not get audio focus.
                 Toast.makeText(RadioService.this, "Cannot play audio.", Toast.LENGTH_SHORT).show();
```

Comment:
Also put the string here into the `strings.xml` file and retrieve it here appropriately. This is a string that'll be updated in the UI and it should be a resource string.